packages/boxel/addon/components/boxel/modal/index.css

```diff
@@ -13,34 +13,44 @@
 }
 
 .boxel-modal {
-  --boxel-modal-offset-top: 30px;
-  --boxel-modal-offset-right: 0;
-  --boxel-modal-offset-left: 0;
-  --boxel-modal-max-width: 1000px;
-
   position: fixed;
   width: 100%;
-  height: calc(100vh - var(--boxel-modal-offset-top));
-  top: var(--boxel-modal-offset-top);
-  left: var(--boxel-modal-offset-left);
-  right: var(--boxel-modal-offset-right);
-  margin: 0 auto;
-  padding: 0;
+  height: calc(100vh - var(--boxel-modal-offset-top, 0px));
+  top: var(--boxel-modal-offset-top, 0px);
+  left: var(--boxel-modal-offset-left, 0px);
+  right: var(--boxel-modal-offset-right, 0px);
+  padding: 0 var(--boxel-sp);
   background: none;
   border: none;
   overflow: hidden;
   z-index: var(--boxel-modal-z-index);
   pointer-events: none;
 }
 
+.boxel-modal--small {
+  --boxel-modal-max-width: 36.25rem; /* 580px */
+}
+
+.boxel-modal--medium {
+  --boxel-modal-max-width: 43.75rem; /* 700px */
+}
+
+.boxel-modal--large {
+  --boxel-modal-offset-top: var(--boxel-sp-lg);
+  --boxel-modal-max-width: 65rem;   /* 1040px */
+}
+
 .boxel-modal__inner {
+  display: flex;
+  align-items: center;
+  justify-content: center;
   width: 100%;
   height: 100%;
-  max-width: var(--boxel-modal-max-width);
-  margin: 0 auto;
-  transition: max-width var(--boxel-transition);
+  margin: auto;
+  max-width: var(--boxel-modal-max-width, 65rem);
```

Comment:
Wondering if this is intentional? There is an escape hatch of sorts for modal sizing by omitting the size and setting the `--boxel-modal-max-width` variable. We could prevent this by setting a size default.